nextflow.config

```
@@ -76,12 +76,11 @@ params {
     deeplc_calibration_set_size     = 0.15
 
     // MultiQC options
-    skip_multiqc                    = false
-    multiqc_config                  = null
-    multiqc_title                   = null
-    multiqc_logo                    = null
-    max_multiqc_email_size          = '25.MB'
-    multiqc_methods_description     = null
+    multiqc_config             = null
+    multiqc_title              = null
+    multiqc_logo               = null
+    max_multiqc_email_size     = '25.MB'
+    multiqc_methods_description = null
```

```suggestion
    multiqc_config              = null
    multiqc_title               = null
    multiqc_logo                = null
    max_multiqc_email_size      = '25.MB'
    multiqc_methods_description = null
```
following https://nf-co.re/docs/contributing/modules#what-is-the-harshil-alignment